lxd/project/project.go

```diff
@@ -58,6 +58,12 @@ func StorageVolume(projectName string, storageVolumeName string) string {
 // name as separate variables.
 func StorageVolumeParts(projectStorageVolumeName string) (string, string) {
 	parts := strings.SplitN(projectStorageVolumeName, "_", 2)
+
+	// If the given name doesn't contain any project, only return the volume name.
+	if len(parts) == 1 {
+		return "", projectStorageVolumeName
```

Comment:
Wondering whether this should return project.Default - but need to understand how this can occur first?

Reply:
I tried to restore some custom storage volumes from PowerFlex that don't have the project's name included (due to the known volume name constraints). This led up to this.

But generally speaking it could happen that you do some storage volume name modifications as part of some recovery that will ultimately cause the same scenario.

Something I was wondering with all of this is that the panic itself wasn't visible in the general LXD daemon log. Is another process running the recovery bits?

Reply:
> But generally speaking it could happen that you do some storage volume name modifications as part of some recovery that will ultimately cause the same scenario.

Any manual modification of storage volume names is invalid - and shouldnt be imported into LXD (although ofc it shouldnt crash either).

So should we detect a an empty project from the volume and not progress?